nova/templates/configmaps/nova.conf.yaml.j2

```diff
@@ -8,8 +8,6 @@ data:
     vif_plugging_is_fatal = False
```

Comment:
Why are we having multiple nova.conf templates for ironic ? Can we have a single common one ?